<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_warehouse">
       <attribute name="title">
        <string>Warehouse</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_search_mode"/>
          </item>
          <item>
           <widget class="QLabel" name="label_search">
            <property name="text">
             <string>Search:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_search"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_warehouse">
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_inventory">
       <attribute name="title">
        <string>Inventory</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="placeholderText">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>NOTICE: Inventory quantity in this view is not guaranteed to be exact. For exact numbers, please contact warehouse manager.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_inventory">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Database"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Item"/>
    <addaction name="actionRemove_Item"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Item"/>
    <addaction name="actionMove_Item"/>
    <addaction name="separator"/>
    <addaction name="actionAdmin"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLog_In"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCompletion_Popup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuUser"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Item"/>
   <addaction name="actionRemove_Item"/>
   <addaction name="separator"/>
   <addaction name="actionChange_Item"/>
   <addaction name="actionMove_Item"/>
   <addaction name="separator"/>
   <addaction name="actionLog_In"/>
   <addaction name="actionLog_Out"/>
   <addaction name="separator"/>
   <addaction name="actionAdmin"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Item">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004260-3d-glossy-blue-orb-icon-arrows-arrow-thick-up.png</normaloff>
     <normalon>:/icons/004298-3d-glossy-blue-orb-icon-arrows-cut-arrow-up.png</normalon>
     <activeon>:/icons/004298-3d-glossy-blue-orb-icon-arrows-cut-arrow-up.png</activeon>:/icons/004260-3d-glossy-blue-orb-icon-arrows-arrow-thick-up.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove_Item">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004257-3d-glossy-blue-orb-icon-arrows-arrow-thick-down.png</normaloff>:/icons/004257-3d-glossy-blue-orb-icon-arrows-arrow-thick-down.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionOpen_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</iconset>
   </property>
   <property name="text">
    <string>Open Database</string>
   </property>
  </action>
  <action name="actionChange_Item">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004298-3d-glossy-blue-orb-icon-arrows-cut-arrow-up.png</normaloff>:/icons/004298-3d-glossy-blue-orb-icon-arrows-cut-arrow-up.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionLog_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</iconset>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</iconset>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</normaloff>:/icons/004254-3d-glossy-blue-orb-icon-arrows-arrow-sparkle.png</iconset>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </action>
  <action name="actionCompletion_Popup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Completion Popup</string>
   </property>
  </action>
  <action name="actionMove_Item">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/004346-3d-glossy-blue-orb-icon-arrows-two-ways-left-right.png</normaloff>:/icons/004346-3d-glossy-blue-orb-icon-arrows-two-ways-left-right.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
